api-js/src/queries/domains/find-domain-by-slug.js

```diff
@@ -40,10 +40,8 @@ const findDomainBySlug = {
     const permitted = await checkDomainPermission(user._id, domain._id, query)
 
     if (!permitted) {
-      console.warn(`User ${user._key} not permitted to access domain.`)
-      throw new Error(
-        `User ${user._key} is not permitted to access specified domain.`,
-      )
+      console.warn(`Could not retrieve domain.`)
```

Comment:
So for the logging messages its alright to keep the user's id, so if they contact the managing team during production they can have some visibility as to which user requested this.